js/controller.js

```diff
@@ -2216,7 +2227,12 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
             this.state.screenToShow = screen;
             this.renderSkin();
           }.bind(this);
-          this.mb.publish(OO.EVENTS.PAUSE);
+          if (doNotPause) {
+            this.pausedCallback();
```

Comment:
This seems a little counterintuitive. Why should you call the callback saying that we paused successfully if we aren't supposed to pause?

Reply:
The ```pausedCallback``` is set right above this line. This is normally called after the player pauses. We now have the concept of a content screen that does not pause the video. To accomplish this, I've modified the ```toggleScreen``` function to not always pause the video. When this is the case, we just immediately call the callback we just defined and set it to null.

Reply:
Ok, sounds good 👍